src/common/wflign/src/wflign_wfa.cpp

```diff
@@ -555,7 +592,7 @@ void do_wfa_patch_alignment(const char *query, const uint64_t &j,
                                              max_distance_threshold);
     }
 
-    const int max_score = (target_length + query_length) * 5;
+    const int max_score = (target_length + query_length);
```

Comment:
In the simulations with low sequence divergence, the inversions reach scores equal to ~`2*(target_length + query_length)`, so it is a good thing to reduce the `max_score`. However, I am not sure if this is enough to patch properly big structural variants or several little ones in the same patched region.